SECURITY.md

```diff
@@ -0,0 +1,40 @@
+# Security Policy
+
+Full details of the Automattic Security Policy can be found on [HackerOne](https://hackerone.com/automattic).
+
+## Supported Versions
+
+Generally, only the latest version of Jetpack has continued support. If a critical vulnerability is found in the current version of Jetpack, we may opt to backport any patches to previous versions. 
+
+## Reporting a Vulnerability
```

Comment:
Should we add a paragraph asking folks to report vulnerabilities at HackerOne, not in this repo? I find this missing from this document.

Reply:
:y: f8a989a76